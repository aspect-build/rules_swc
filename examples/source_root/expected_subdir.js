export var a = "a";

//# sourceMappingURL=subdir.js.map