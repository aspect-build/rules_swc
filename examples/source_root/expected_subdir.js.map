{"version":3,"sources":["subdir.ts"],"sourceRoot":"../../../debug/source","sourcesContent":["export const a: string = \"a\";\n"],"names":["a"],"mappings":"AAAA,OAAO,IAAMA,IAAY,IAAI"}