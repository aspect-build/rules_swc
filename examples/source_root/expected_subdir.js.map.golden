{
    "sources": [
        "src/subdir.ts"
    ],
    "sourceRoot": "../../../debug/source"
}